_docs/node-operator-requirements-service.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Node Operator Requirements via partner services
+subtitle: How to setup your node with one of our node infrastructure partner services
+author: irsal
+tags: [infra]
+---
+
+## Node Infrastructure Requirements
+Since we've vetted these services, you can get many of the following with the out-of-the-box configurations that we've setup.
+- Location: non-European countries
+- At least 8 CPU cores.
+- At least 16GB of RAM.
+- At least 1TB of storage.
+- Monitoring and alerting in place (Prometheus, Grafana, the like).
+
+Please sync a few days before your intended collation / block production candidacy to sync the nodes. Once your node is synced, you will find that the block numbers are up to date with both the relay chain and the parachain. Check out the [Telemetry](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d) site for more information.
+
+----------------------------------------------------------------------------------------------------------------------------------
+
+## Neumann Network - Testnet
+> Neumann is a testnet, thus it will have maximal amount of chaos and will NOT have 100% uptime. This might be reset at a given time dur to a number of different reasons. If you are a stakeholder (collator, dev, consumer) of this chain, please expect occassional instability, thus the ability to re-deploy your system as needed. The OAK team will always re-instate the same wallet amounts as before a reset occurs, so users should expect that for any given chain reset. If a chain reset is required, it will be announced via Discord.
+
+- [Neumann Chain Spec](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/node/res/neumann.json)
+- [OAK-hosted Rococo Chain Spec](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/node/res/neumann-rococo-testnet.json)
+
+## Turing Network
+- [Turing Chain Spec](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/node/res/turing.json)
+- [Kusama Chain Spec](https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json)
+
+----------------------------------------------------------------------------------------------------------------------------------
+
+## OnFinality
+
+### Step 1: Setup your OnFinality account
+Create an [OnFinality account](https://app.onfinality.io/signup), and add a payment method to your [Accounts](https://app.onfinality.io/account).
+
+### Step 2: Dedicated Node selection
+Once you've confirmed your account, you are ready to setup a node. Starting with navigating to the Dedicated Nodes section. 
+
+![onfinality-setup-1](../assets/img/node-operator/onfinality-setup-1.png)
```

Comment:
None of these, pics are showing up. 
![image](https://user-images.githubusercontent.com/16072671/168698855-5ce1bb8f-c53d-4eb9-baed-b8b1062d3120.png)

Reply:
Looks like path might need one more `../`

Reply:
thank you!